dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -565,7 +565,7 @@ public String[] tryExecuteSqlResolveColumnNames(DbType sourceType, BaseConnectio
             int num = md.getColumnCount();
             columnNames = new String[num];
             for (int i = 1; i <= num; i++) {
-                columnNames[i - 1] = md.getColumnName(i);
+                columnNames[i - 1] = md.getColumnName(i).replaceAll("t.","");
```

Comment:
I'm not familiar with datax. Can you describe the usage of this change?

Reply:
Because HiveStatementParser using druid uses 'SELECT t.* FROM ( %s ) t WHERE 0 = 1' to parse the column name with a temporary table （t.a,t.b）.so i need replace it

Reply:
If so, we should judge whether is hive type before performing this replacement.

Reply:
'SELECT t.* FROM ( %s ) t WHERE 0 = 1' This statement will execute all jdbc types, that's why I put the repalce statement here @SbloodyS

Reply:
cc @zhuangchong